features/lr_casework_search_for_deed.feature

```diff
@@ -32,11 +32,54 @@ And I create the deed via the Deed API
 When I search the deed store with title_number "DM11"
 Then I verify the returned deed information
 
-
 @us152
 Scenario: Error is returned when search returns no deed
 Given I setup a deed with <1> borrowers
 And I amend "title_number" to "DM11"
 And I create the deed via the Deed API
-When I search the deed store with title_number "99999"
+When I search the deed store with title_number "14789"
 Then a status code of "404" is returned
+
+# List of possible deed status'  found in dm-deed-api/application/deed/deed_status.py
+# "DRAFT"
+# "PARTIALLY-SIGNED"
+# "SUPERCEDED"
+# "ABORTED"
+# "STOPPED"
+# "ALL-SIGNED"
+# "EFFECTIVE"
+# "SUBMITTED"
+# "STORED"
+# "REGISTERED"
+
+@us133f @us152
+Scenario: Verify deed status is All-signed when fully signed
+Given I setup a deed with <1> borrowers
+And I amend "title_number" to "99999"
+When I create the deed
+And I search the deed store with title_number "99999"
+Then I verify the returned deed information is "DRAFT"
+And borrower <1> views the deed
+And the borrower <1> signature element is present on page
+Then the deed is digitally signed by borrower <1>
+When I search the deed store with title_number "99999"
+Then I verify the returned deed information is "ALL-SIGNED"
+
+@us133f @us152
+Scenario: Verify deed status is PARTIALLY-SIGNED when not fully signed
```

Comment:
Feel like these scenarios could do with a rename just to make it clearer what partially signed and fully signed is. So maybe something like 'Verify deed status is PARTIALLY SIGNED when only one of two borrowers have signed the deed/all borrowers have signed the deed' or something like that perhaps.